### DISCUSSION TITLE
Postmark / External SMTP server integration

### DISCUSSION THREAD

**Discussion by maticboncina** [simple-login/app#1032] [Feature Requests]
It would be cool if we could use external SMTP services such as postmark or amazon ses to send forwarded emails. It's much more convenient to use a solution especially if you have many users on owned installation and are worried about deliverability and email server management.

Simply there could be a field entry form, where the admin could enter credentials for the remote SMTP service to be used overwriting the default sender. To be clear I am not asking to get access to SimpleLogin's SMTP server more so to use external service.

**Comment by nguyenkims** [comment]
@TitanBlast we have a guide on how to use AWS SES to send out emails https://github.com/simple-login/app/blob/01bf037638429a982a196123911133bef29c3d08/docs/ses.md#L1-L0

**Comment by maticboncina** [comment]
Thank you :) If I can somehow close this, please do let me know :)
